### PR TITLE
add rbac for application profile CRDs

### PR DESCRIPTION
**Pull request by matthyx**
## PR Type:
Enhancement

___
## PR Description:
This PR enhances the Role-Based Access Control (RBAC) for application profile CRDs by adding new resources to the cluster role. It also introduces a new configuration for enabling the application profile service. Additionally, the PR updates the image repositories and tags for the storage and node-agent services to use a different source.

___
## PR Main Files Walkthrough:
`charts/kubescape-operator/templates/_helpers.tpl`: Added a new helper for runtime observability configuration.
`charts/kubescape-cloud-operator/README.md`: Removed the content of the README file, indicating the deprecation of the Kubescape Operator.
`charts/kubescape-operator/templates/node-agent/clusterrole.yaml`: Added new resources ("applicationactivities", "applicationprofiles", "applicationprofilesummaries") to the cluster role for the node-agent.
`charts/kubescape-operator/templates/node-agent/configmap.yaml`: Added a new configuration for enabling the application profile service in the node-agent's configmap.
`charts/kubescape-operator/values.yaml`: Updated the image repositories and tags for the storage and node-agent services. Also, added a new configuration for runtime observability.

### CODE REVIEW

File: charts/kubescape-operator/values.yaml [hunk 1]
```diff
@@ -506,8 +506,8 @@ storage:
 
   replicaCount: 1
   image:
-    repository: quay.io/kubescape/storage
-    tag: v0.0.18
+    repository: quay.io/matthiasb_1/storage
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:509]: Please use the kubescape org

**Review by dwertent — CHANGES_REQUESTED** [review]

File: charts/kubescape-operator/values.yaml (hunk 1 above)
Comment (matthyx) [reply charts/kubescape-operator/values.yaml:509]: yes, I'll update the node-agent too

**Comment by dwertent** [comment]
@matthyx can we merge this? what is the status?

**Comment by matthyx** [comment]
> @matthyx can we merge this? what is the status?

yes, it just adds some rbac along with the image bumps for application profile

**Review by dwertent — APPROVED** [review]

**Pull request merged**